### ISSUE TITLE
No separate package for green_sardine

### ISSUE THREAD

**Issue by ivan-volnov** [freebsd/drm-kmod-firmware#24]
Hello,

There is no separate package for green_sardine:

```
New packages to be INSTALLED:
	gpu-firmware-amd-kmod-banks: 20220511
	gpu-firmware-amd-kmod-bonaire: 20220511
	gpu-firmware-amd-kmod-carrizo: 20220511
	gpu-firmware-amd-kmod-fiji: 20220511
	gpu-firmware-amd-kmod-hainan: 20220511
	gpu-firmware-amd-kmod-hawaii: 20220511
	gpu-firmware-amd-kmod-kabini: 20220511
	gpu-firmware-amd-kmod-kaveri: 20220511
	gpu-firmware-amd-kmod-mullins: 20220511
	gpu-firmware-amd-kmod-navi10: 20220511
	gpu-firmware-amd-kmod-navi12: 20220511
	gpu-firmware-amd-kmod-navi14: 20220511
	gpu-firmware-amd-kmod-oland: 20220511
	gpu-firmware-amd-kmod-picasso: 20220511
	gpu-firmware-amd-kmod-pitcairn: 20220511
	gpu-firmware-amd-kmod-polaris10: 20220511
	gpu-firmware-amd-kmod-polaris11: 20220511
	gpu-firmware-amd-kmod-polaris12: 20220511
	gpu-firmware-amd-kmod-raven: 20220511
	gpu-firmware-amd-kmod-raven2: 20220511
	gpu-firmware-amd-kmod-renoir: 20220511
	gpu-firmware-amd-kmod-si58: 20220511
	gpu-firmware-amd-kmod-stoney: 20220511
	gpu-firmware-amd-kmod-tahiti: 20220511
	gpu-firmware-amd-kmod-tonga: 20220511
	gpu-firmware-amd-kmod-topaz: 20220511
	gpu-firmware-amd-kmod-vega10: 20220511
	gpu-firmware-amd-kmod-vega12: 20220511
	gpu-firmware-amd-kmod-vega20: 20220511
	gpu-firmware-amd-kmod-vegam: 20220511
	gpu-firmware-amd-kmod-verde: 20220511
	gpu-firmware-intel-kmod-broxton: 20220511
	gpu-firmware-intel-kmod-coffeelake: 20220511
	gpu-firmware-intel-kmod-elkhartlake: 20220511
	gpu-firmware-intel-kmod-geminilake: 20220511
	gpu-firmware-intel-kmod-icelake: 20220511
	gpu-firmware-intel-kmod-kabylake: 20220511
	gpu-firmware-intel-kmod-rocketlake: 20220511
	gpu-firmware-intel-kmod-skylake: 20220511
	gpu-firmware-intel-kmod-tigerlake: 20220511
	gpu-firmware-radeon-kmod-aruba: 20220511
	gpu-firmware-radeon-kmod-barts: 20220511
	gpu-firmware-radeon-kmod-bonaire: 20220511
	gpu-firmware-radeon-kmod-btc: 20220511
	gpu-firmware-radeon-kmod-caicos: 20220511
	gpu-firmware-radeon-kmod-cayman: 20220511
	gpu-firmware-radeon-kmod-cedar: 20220511
	gpu-firmware-radeon-kmod-cypress: 20220511
	gpu-firmware-radeon-kmod-hainan: 20220511
	gpu-firmware-radeon-kmod-hawaii: 20220511
	gpu-firmware-radeon-kmod-juniper: 20220511
	gpu-firmware-radeon-kmod-kabini: 20220511
	gpu-firmware-radeon-kmod-kaveri: 20220511
	gpu-firmware-radeon-kmod-mullins: 20220511
	gpu-firmware-radeon-kmod-oland: 20220511
	gpu-firmware-radeon-kmod-palm: 20220511
	gpu-firmware-radeon-kmod-pitcairn: 20220511
	gpu-firmware-radeon-kmod-r100: 20220511
	gpu-firmware-radeon-kmod-r200: 20220511
	gpu-firmware-radeon-kmod-r300: 20220511
	gpu-firmware-radeon-kmod-r420: 20220511
	gpu-firmware-radeon-kmod-r520: 20220511
	gpu-firmware-radeon-kmod-r600: 20220511
	gpu-firmware-radeon-kmod-r700: 20220511
	gpu-firmware-radeon-kmod-redwood: 20220511
	gpu-firmware-radeon-kmod-rs600: 20220511
	gpu-firmware-radeon-kmod-rs690: 20220511
	gpu-firmware-radeon-kmod-rs780: 20220511
	gpu-firmware-radeon-kmod-rv610: 20220511
	gpu-firmware-radeon-kmod-rv620: 20220511
	gpu-firmware-radeon-kmod-rv630: 20220511
	gpu-firmware-radeon-kmod-rv635: 20220511
	gpu-firmware-radeon-kmod-rv670: 20220511
	gpu-firmware-radeon-kmod-rv710: 20220511
	gpu-firmware-radeon-kmod-rv730: 20220511
	gpu-firmware-radeon-kmod-rv740: 20220511
	gpu-firmware-radeon-kmod-rv770: 20220511
	gpu-firmware-radeon-kmod-sumo: 20220511
	gpu-firmware-radeon-kmod-sumo2: 20220511
	gpu-firmware-radeon-kmod-tahiti: 20220511
	gpu-firmware-radeon-kmod-turks: 20220511
	gpu-firmware-radeon-kmod-verde: 20220511

Installed packages to be UPGRADED:
	gpu-firmware-kmod: g20210330 -> 20220511,1
```

I'm on 13.1-RC5 still. Does it depend on Release version?
Just want to ensure my update will be fine.

**Comment by evadot** [comment]
Official packages are still built on 13.0 so there won't be no green_sardine and sienna_cichlid packages for ~3 months.
It's only available as ports for 13.1 users.

**Comment by ivan-volnov** [comment]
Ok. Got it. Thanks!
